### PR TITLE
Fixing cmake to use conan version of RivieraSDK

### PR DESCRIPTION
**Pull request by Michael-Zeimbekakis-Bose**

### CODE REVIEW

File: Bose/.gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ conaninfo.txt
 activate_run.sh
 deactivate_run.sh
 graph_info.json
+LibMultiThread/builds
```
Comment (Michael-Cook-Bose) [inline Bose/.gitignore]: Please add the final newline (and fix your editor :-))

Comment (Michael-Zeimbekakis-Bose) [reply Bose/.gitignore]: Done

**Comment by Mark-Hatch-Bose** [comment]
Any reason we added the txt generator @Michael-Zeimbekakis-Bose, should we explicitly add in MakefilePackage

**Comment by Michael-Zeimbekakis-Bose** [comment]
@Mark-Hatch-Bose We dropped the cmake generator with the shift to OpenSourcePackage, so it needed to be added back. The txt generator was already in OpenSourcePackage and I didn't want to override it, so I added it to this specific conanfile.

**Pull request merged**